elytron/WFCORE-5120-client-default-ssl-context.adoc

```diff
@@ -0,0 +1,86 @@
+= WFCORE-5120 Automatic registration of client side / JVM wide default SSLContext
+:author:            Diana Vilkolak Krepinska
+:email:             dvilkola@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This issue is about providing ability to register a client side JVM wide default SSLContext for the use of any libraries that support the use of the default SSL context.
+The loading of elytron client configuration should happen with minimal configuration from the user in cases where API is not used directly.
```

Comment:
How is the Elytron client configuration involved? What information is this sentence bringing out to the reader? Can you describe better/make an example of the "not using the API directly" use case?